openslide/__init__.py

```diff
@@ -359,11 +359,12 @@ def read_region(self, location, level, size):
         # the image.  Create a transparent tile of the correct size and
         # paste the valid part of the region into the correct location.
         image_topleft = [
-            max(0, min(l, limit - 1)) for l, limit in zip(location, self._image.size)
+            max(0, min(offset, limit - 1))
+            for offset, limit in zip(location, self._image.size)
         ]
         image_bottomright = [
-            max(0, min(l + s - 1, limit - 1))
-            for l, s, limit in zip(location, size, self._image.size)
+            max(0, min(offset + s - 1, limit - 1))
+            for offset, s, limit in zip(location, size, self._image.size)
```

Comment:
This seems like a strange change.  `l` is for location, just as the unchanged `s` is for `size`.  We're introducing inconsistent terminology here.

Reply:
ah, location. flake complained about the single letter variable and I wasn't sure what 'l' meant, I can change that to location